h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -457,10 +457,14 @@ private static boolean standardize(double sigma) {
   /** The {@code Vec.byteSize} of all Vecs
    *  @return the {@code Vec.byteSize} of all Vecs */
   public long byteSize() {
-    Vec[] vecs = bulkRollups();
-    long sum=0;
-    for (Vec vec : vecs) sum += vec.byteSize();
-    return sum;
+    try {
+      Vec[] vecs = bulkRollups();
+      long sum = 0;
+      for (Vec vec : vecs) sum += vec.byteSize();
+      return sum;
+    } catch(RuntimeException ex) {
```

Comment:
Frame was deleted from the DKV, but the Log.info(Frame) in the leaked key check was asking for byteSize, but the Rollups failed with an exception.